src/projetos/ProjetoService.ts

```
@@ -28,6 +29,14 @@ class ProjetoService extends BaseService {
   async remove(id: number): Promise<void> {
     return this.delete(`/${id}`)
   }
+
+  async patch<T = any>(url: string, data?: any): Promise<T> {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)